src/Core/src/Handlers/Image/ImageHandler.Android.cs

```diff
@@ -47,8 +47,14 @@ public static void MapBackground(IImageHandler handler, IImage image)
 		public static void MapSource(IImageHandler handler, IImage image) =>
 			MapSourceAsync(handler, image).FireAndForget(handler);
 
-		public static Task MapSourceAsync(IImageHandler handler, IImage image) =>
-			handler.SourceLoader.UpdateImageSourceAsync();
+		public static Task MapSourceAsync(IImageHandler handler, IImage image)
+		{
+			var result = handler.SourceLoader.UpdateImageSourceAsync();
+
+			result.ContinueWith((action) => handler.PlatformView.UpdateIsAnimationPlaying(image));
```

Comment:
To launch the animation, the ImageView` Drawable `is required, which will be null until the image is loaded. The Mapper invokes the logic to update `IsAnimationPlaying` before loading the image and with a null `Drawable` nothing was done. Now, after loading the image, we invalidate the changes in the `IsAnimationPlaying` property.

Reply:
Maybe this would be easier as async? Maybe this all should be ending with FireAndForget @PureWeen ? I feel like this method is wrong before with the async things.

This is button handler on windows:


		public static void MapImageSource(IButtonHandler handler, IImage image) =>
			handler
				.ImageSourceLoader
				.UpdateImageSourceAsync()
				.FireAndForget(handler);

I am also scared because looking at the code for the UpdateImageSourceAsync method, it has an await with ConfigureAwait(false) which means it may return on a non-ui thread causing an exception in the next part:

    result.ContinueWith((action) => handler.PlatformView.UpdateIsAnimationPlaying(image));

I am thinking maybe we should remove that ConfigureAwait as well to be safe, then use the ContinueWith and then FireAndForget?